test/integration/forms/form.js

```diff
@@ -1225,6 +1225,119 @@ context('Patient Action Form', function() {
       .should('contain', 'Test Field')
       .should('contain', 'Test field widget');
   });
+
+  specify('save and go back button', function() {
+    localStorage.setItem('form-state_11111', JSON.stringify({ saveButtonType: 'saveAndGoBack' }));
+
+    cy
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.form.data = { id: '11111' };
+        fx.data.relationships['program-action'] = { data: { id: '11111' } };
+        fx.data.relationships['form-responses'].data = [
+          { id: '1', meta: { created_at: testTs() } },
+        ];
+
+        return fx;
+      })
+      .routeForm(_.identity, '11111')
+      .routeFormDefinition()
+      .routeFormActionFields()
+      .routeFormResponse(fx => {
+        fx.data.storyTime = 'Once upon a time...';
+
+        return fx;
+      })
+      .routeActionActivity()
+      .routePatientByAction()
+      .visit('/patient-action/1/form/11111')
+      .wait('@routeAction')
+      .wait('@routeForm')
+      .wait('@routePatientByAction')
+      .wait('@routeFormDefinition')
+      .wait('@routeFormResponse');
+
+    cy
+      .get('.form__controls')
+      .find('button')
+      .contains('Update')
+      .click()
+      .wait('@routeFormActionFields');
```

Comment:
Is there a way this test can work without doing the clicking of the `Update` button first? I tried skipping that, but it was causing all sorts of errors where elements on the page were being "not attached to the DOM".

Also looking for any other unnecessary things to trim from these tests I added.

Reply:
yeah if you remove the `form-responses` relation it won't be needed